packages/cozy-client/src/CozyClient.js

```diff
@@ -164,37 +159,11 @@ export default class CozyClient {
   }
 
   async save(document, mutationOptions = {}) {
-    const ret = await this.validate(document)
+    const ret = await this.schema.validate(document)
     if (ret !== true) throw new Error('Validation failed')
     return this.mutate(this.getDocumentSavePlan(document), mutationOptions)
   }
 
-  async validate(document) {
```

Comment:
Just a head's up that this will also be a breaking change (cozy-photo uses `client.validate`)

Reply:
Done in https://github.com/cozy/cozy-client/pull/158/commits/9c9d061a5adb9be8cd363795f748cdcf633827db

Reply:
Cool, thanks!